.husky/pre-commit

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-# Run pretty-quick to format staged files
-npx pretty-quick --staged
-
```

Comment:
I removed this because we're no longer using Prettier, but wasn't sure how best to replace this